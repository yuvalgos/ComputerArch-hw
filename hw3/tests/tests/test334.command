./dflow_calc tests/test334.dat tests/test334.in d141 p156 p11 d76 p38 d99 p165 d37 p7 d24 d162 p60 p69 p116 d140 d106 p18 d128 d134 p80 d54 d15 d48 p77 d106 d59 p139 d131 d81 p96 p135 d179 p127 p189 d155 p171 d177 d56 p178 p98 p157 d152 p137 p181 p97 p149 p128 d70 p56 p106