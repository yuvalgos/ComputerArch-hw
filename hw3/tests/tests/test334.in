7 4 20 16
7 11 21 1
6 18 22 29
5 6 5 24
5 18 2 29
0 22 18 4
4 2 7 9
4 15 19 9
8 15 23 20
5 16 11 26
7 17 4 0
2 28 5 6
8 3 30 7
2 11 29 22
8 9 3 7
6 10 15 28
8 12 19 24
6 15 9 17
5 2 11 24
2 12 28 8
3 29 12 0
3 24 2 20
8 12 8 9
3 10 19 15
3 25 30 30
5 30 10 19
4 12 18 16
8 2 10 13
4 27 11 8
5 6 28 28
1 16 27 22
8 20 21 26
5 14 2 18
1 25 8 27
7 23 26 15
0 22 13 21
2 11 24 17
6 20 27 2
2 27 22 27
6 24 17 12
2 1 22 24
1 18 23 8
4 12 25 24
0 26 9 4
0 4 10 11
7 13 1 17
7 24 24 9
2 13 7 11
0 3 27 13
3 25 31 29
8 28 22 29
7 19 29 19
2 16 10 19
0 1 9 25
5 4 2 30
7 16 28 13
6 17 20 9
2 1 29 7
0 25 10 4
1 24 15 6
5 31 31 6
7 3 0 7
5 14 23 12
5 30 4 5
0 8 2 23
7 9 7 19
1 26 16 9
0 8 6 23
8 29 9 7
2 21 14 13
2 30 12 8
7 19 13 25
6 9 0 6
7 4 13 27
3 4 26 16
3 21 1 16
7 13 2 9
0 31 4 1
2 9 8 3
4 3 28 28
0 31 25 11
4 19 5 20
3 29 17 26
4 3 30 15
1 21 24 27
6 31 14 0
2 29 12 0
4 9 14 15
1 29 15 8
4 0 24 17
5 27 28 17
0 4 3 20
3 23 7 24
4 11 27 21
2 29 25 20
2 22 21 29
0 14 12 22
7 21 23 3
3 16 28 21
8 0 21 15
1 20 14 26
4 6 29 5
7 21 14 27
8 0 18 30
6 13 30 12
6 12 12 26
8 24 4 2
5 17 2 11
0 12 12 30
7 20 29 26
7 19 27 30
8 11 0 29
4 6 30 28
4 14 23 0
6 31 8 9
5 17 29 6
8 9 19 28
1 6 17 14
0 19 12 7
4 4 20 25
1 15 23 29
6 4 9 14
0 23 14 23
1 20 21 26
6 20 19 15
2 1 0 23
8 12 21 0
5 28 12 5
5 1 29 16
3 29 10 24
3 16 13 22
2 6 1 17
1 5 12 27
4 28 16 18
7 17 2 1
8 18 28 24
2 23 26 28
0 9 6 18
7 29 27 26
1 18 28 28
4 0 19 0
3 3 5 28
5 22 28 11
5 27 12 8
5 13 26 6
8 23 31 24
4 2 17 25
4 19 29 27
4 4 1 25
2 5 23 17
0 8 23 12
5 15 28 19
4 16 21 27
5 11 30 23
2 24 22 27
8 15 9 5
7 20 31 18
6 30 3 11
2 10 9 3
2 30 16 24
0 5 11 20
3 8 9 14
4 14 24 12
0 4 14 3
5 28 11 4
6 24 2 22
3 11 9 30
0 13 28 28
1 25 23 1
1 15 11 1
7 8 13 3
4 26 21 31
0 23 0 31
6 4 14 30
4 3 23 11
4 20 18 31
5 6 14 23
8 4 9 0
3 28 4 16
5 0 28 7
5 17 5 21
8 15 12 12
0 26 17 11
5 5 14 25
3 15 10 14
0 24 6 21
4 17 7 22
8 19 25 26
5 1 5 0
7 25 0 21
4 30 27 12
3 3 26 6
8 7 3 11
6 4 29 27
5 20 26 5
7 16 24 22
8 29 28 7
2 0 3 3
8 25 24 31
4 12 23 22
