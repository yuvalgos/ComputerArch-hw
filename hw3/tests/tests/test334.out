Reading the opcodes latency info from tests/test334.dat ... Got latency for 9 opcodes
Reading the program file tests/test334.in ... Found 200 instructions
getProgDepth()==277
getInstDeps(141)=={132,133}
getDepDepth(156)==141
getDepDepth(11)==13
getInstDeps(76)=={27,72}
getDepDepth(38)==37
getInstDeps(99)=={97,17}
getDepDepth(165)==169
getInstDeps(37)=={28,27}
getDepDepth(7)==0
getInstDeps(24)=={-1,-1}
getInstDeps(162)=={154,126}
getDepDepth(60)==31
getDepDepth(69)==95
getDepDepth(116)==159
getInstDeps(140)=={118,103}
getInstDeps(106)=={91,27}
getDepDepth(18)==26
getInstDeps(128)=={94,98}
getInstDeps(134)=={27,128}
getDepDepth(80)==108
getInstDeps(54)=={27,25}
getInstDeps(15)=={8,11}
getInstDeps(48)=={38,47}
getDepDepth(77)==85
getInstDeps(106)=={91,27}
getInstDeps(59)=={17,29}
getDepDepth(139)==138
getInstDeps(131)=={128,115}
getInstDeps(81)=={-1,37}
getDepDepth(96)==120
getDepDepth(135)==138
getInstDeps(179)=={178,-1}
getDepDepth(127)==182
getDepDepth(189)==226
getInstDeps(155)=={137,149}
getDepDepth(171)==159
getInstDeps(177)=={137,140}
getInstDeps(56)=={37,14}
getDepDepth(178)==206
getDepDepth(98)==119
getDepDepth(157)==192
getInstDeps(152)=={102,143}
getDepDepth(137)==161
getDepDepth(181)==182
getDepDepth(97)==100
getDepDepth(149)==158
getDepDepth(128)==126
getInstDeps(70)=={42,67}
getDepDepth(56)==110
getDepDepth(106)==112
